.gradle/buildOutputCleanup/cache.properties

```diff
@@ -1,2 +1,2 @@
-#Sat Dec 05 22:32:26 EET 2020
-gradle.version=6.5
+#Thu Dec 24 13:00:37 EET 2020
```

Comment:
As per our Telegram chat, please remove this file. It's not supposed to be in the VCS.

Reply:
The same comment concerns all the files from the `.gradle` directory.